cmd/minikube/cmd/status.go

```diff
@@ -101,47 +80,34 @@ var statusCmd = &cobra.Command{
 			apiserverSt, err = clusterBootstrapper.GetAPIServerStatus(ip, apiserverPort)
 			if err != nil {
 				glog.Errorln("Error apiserver status:", err)
-			} else if apiserverSt != state.Running.String() {
-				returnCode |= clusterNotRunningStatusFlag
 			}
 
 			ks, err := pkgutil.GetKubeConfigStatus(ip, util.GetKubeConfigPath(), config.GetMachineName())
 			if err != nil {
 				glog.Errorln("Error kubeconfig status:", err)
 			}
+
 			if ks {
 				kubeconfigSt = "Correctly Configured: pointing to minikube-vm at " + ip.String()
 			} else {
 				kubeconfigSt = "Misconfigured: pointing to stale minikube-vm." +
 					"\nTo fix the kubectl context, run minikube update-context"
-				returnCode |= k8sNotRunningStatusFlag
 			}
-		} else {
-			returnCode |= minikubeNotRunningStatusFlag
 		}
 
-		status := Status{
-			Host:       hostSt,
-			Kubelet:    kubeletSt,
-			APIServer:  apiserverSt,
-			Kubeconfig: kubeconfigSt,
-		}
-		tmpl, err := template.New("status").Parse(statusFormat)
-		if err != nil {
-			exit.WithError("Error creating status template", err)
-		}
-		err = tmpl.Execute(os.Stdout, status)
-		if err != nil {
-			exit.WithError("Error executing status template", err)
-		}
+		var data [][]string
+		data = append(data, []string{pkg_config.GetMachineName(), hostSt, kubeletSt, apiserverSt, kubeconfigSt})
 
-		os.Exit(returnCode)
+		table := tablewriter.NewWriter(os.Stdout)
+		table.SetHeader([]string{"Profile", "Host", "Kubelet", "APIServer", "Kubectl"})
```

Comment:
Use lowercase for `kubelet` and `kubectl`. If it looks better for consitency, you can lowercase all of them.

My only other concern for this approach is that adding more checks may consider rethinking of the layout. For instance, checks for coredns, or internet connectivity.

Altogether, this PR is an improvement in the right direction, IMHO.